spec/integration/compose/expected_volumes_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "spec_helper"
+
+RSpec.describe "Expected volumes" do
+  ProjectsHelper.rails_projects.each do |project_name|
```

Comment:
Nice test - good to pick up the other volumes to 💯.

I found the helper quite confusing, though:

- There's now a bit of a cycle between `ProjectsHelper` and `ComposeHelper`.
- The `rails_projects` is only subtly different to `rails_app-services`.

An alternative might be to have `rails_projects` inline with the test, like we do [the rather hairy makefile test](https://github.com/alphagov/govuk-docker/blob/master/spec/integration/make/dependencies_spec.rb). It's been a tricky task working out what helpers to have and how to compose them. Having Rails projects computed inline would at least reduce the effort digging in to the call stack.

Reply:
Sure, I've switched both `rails_projects` and `node_projects` to inline. We can deal with specificities if the helpers are needed elsewhere. Since the methods need to be available outside of an example they're sensitive to ordering.